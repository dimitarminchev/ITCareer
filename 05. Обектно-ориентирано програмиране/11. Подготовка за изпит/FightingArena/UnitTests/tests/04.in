HeroInfo:Gosho
CreateHero:Priest:Pesho
HeroInfo:Pesho
CreateWeapon:Pesho:MagicWand:NameOfWeapon:5:5:5
HeroInfo:Pesho
CloseArena