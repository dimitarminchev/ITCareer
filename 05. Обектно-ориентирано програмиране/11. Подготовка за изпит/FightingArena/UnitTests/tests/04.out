Hero with name: Gosho does not exist!
Priest: Pesho joined the Arena!
Priest: Pesho - Level: 0
Experience: 0
Power: 0.00
Successfully armed hero Pesho with weapon MagicWand!
Priest: Pesho - Level: 0
Experience: 0
MagicWand: NameOfWeapon
  *Strength: 5
  *Agility: 5
  *Intelligence: 25
Power: 93.33
Heroes: 1
Priest: Pesho - Level: 0
Experience: 0
MagicWand: NameOfWeapon
  *Strength: 5
  *Agility: 5
  *Intelligence: 25
Power: 93.33